src/NServiceBus.AcceptanceTesting/Support/RunSettings.cs

```diff
@@ -7,15 +7,19 @@ namespace NServiceBus.AcceptanceTesting.Support
 
     public class RunSettings : IEnumerable<KeyValuePair<string, object>>
     {
-        public RunSettings()
+        public TimeSpan? TestExecutionTimeout
```

Comment:
What breaking change  @andreasohlund ?

It seems it was changed from `TimeSpan` to `TimeSpan?`, a recompile will fix this without any code changes